docs/examples.md

```diff
@@ -76,6 +76,26 @@ $y$j9T$A0Y3wwVOKP69S.1K/zYGN.$S596l11UGH3XjN...
 
 The `yescrypt` hashing method is recommended for new passwords. For more details on hashing methods, see `man 5 crypt`.
 
+In case inline definitions of SSH keys are undesirable you can use file references instead. This example creates three users (`core`, `jlebon`, `miabbott`) and sets up their respective SSH keys via file references instead of inlining them. This requires using `butane` with the `--files-dir` parameter to allow loading files from disk.
```

Comment:
This is being put in the password authentication subsection where it doesn't really belong.  Maybe let's add a new subsection?

For content, maybe something like:

```suggestion
In addition to defining SSH keys directly in the config, you can embed keys from local files at transpile time. This example creates a `core` user and configures its SSH keys from local files. The file paths are relative to the directory specified with the `--files-dir` command-line option, which must be provided.
```

Reply:
You're absolutely right. It wasn't actually my intention to put it into the password section.